src/ast/seq_decl_plugin.cpp

```diff
@@ -1190,12 +1190,15 @@ unsigned seq_util::max_mul(unsigned x, unsigned y) const {
     return (r > UINT_MAX) ? UINT_MAX : (unsigned)r;
 }
 
+bool seq_util::is_any_const_char(expr* e) const {
```

Comment:
just call this "is_const_char". 
We rely on overloading in many other similar cases.